lnst/Recipes/ENRT/CTInsertionRateNftablesRecipe.py

```diff
@@ -0,0 +1,18 @@
+from .SimpleNetworkRecipe import SimpleNetworkRecipe
+from .ConfigMixins.NftablesConntrackMixin import NftablesConntrackMixin
+
+from lnst.Common.Parameters import ConstParam
+from lnst.RecipeCommon.Perf.Evaluators import NonzeroFlowEvaluator
+
+
+class CTInsertionRateNftablesRecipe(NftablesConntrackMixin, SimpleNetworkRecipe):
```

Comment:
I'm a bit confused about this class name. It says "insertion rate", but it does not measure anything related to rule insertion - could you please clarify?

Maybe a documentation would help me to understand what this recipe does, so please add a docstring.

Reply:
Thanks, looks ok now.